README.md

```diff
@@ -47,12 +47,12 @@ target_link_libraries(my_library cuco)
 
 This will take care of downloading `cuCollections` from GitHub and making the headers available in a location that can be found by CMake. Linking against the `cuco` target will provide everything needed for `cuco` to be used by the `my_library` target.
 
-<a name="link-footnote">1</a>: `cuCollections` is header-only and therefore there is no binary component to "link" against. The linking terminology comes from CMake's `target_link_libraries` which is still used even for header-only library targets. 
+<a name="link-footnote">1</a>: `cuCollections` is header-only and therefore there is no binary component to "link" against. The linking terminology comes from CMake's `target_link_libraries` which is still used even for header-only library targets.
 
 ## Requirements
-- `nvcc 11+`
+- `nvcc 11.6+`
```

Comment:
This is the only relevant change in this file.

Reply:
@PointKernel if libcudf still supports nvcc 11.5 we can choose that as the minimum version.

Reply:
Had to revert to 11.5 since one of our CI runners is still using this version.